<launch>
    <node pkg="joy" type="joy_node" name="joy_node" output="screen">
      <param name="autorepeat_rate" value="10" />
	  <param name="dev" value="/dev/input/js0" />
    </node>
    <node pkg="ardrone_joystick" type="ardrone_teleop" name="ardrone_joystick" output="screen">
	  <remap from="/cmd_vel" to="/ar_pid/cmd_vel"/>
    </node>
	<node pkg="ar_tracking" type="ar_pid" name="ar_tracking" output="screen">
	  
	</node>
</launch>
